packages/apps-config/src/ui/general/index.ts

```diff
@@ -33,7 +34,8 @@ const chainColors: Record<string, any> = [
 const nodeColors: Record<string, any> = [
   ['centrifuge chain', nodeCentrifuge],
   ['edgeware node', nodeEdgeware],
-  ['nodle chain node', nodeNodle]
+  ['nodle chain node', nodeNodle],
+  ['datahighway node', nodeDataHighway]
```

Comment:
Alphabetical order - these things have the tendency to grow, so just easier to maintain.